app/services/doc_auth/socure/requests/document_request.rb

```diff
@@ -27,7 +27,7 @@ def lang(language)
 
         def body
           redirect = {
-            method: 'POST',
+            method: 'GET',
```

Comment:
curious, why we're switching to GET here?

Reply:
https://gsa-tts.slack.com/archives/C05J3KMPSHE/p1730933390138349?thread_ts=1730926185.120449&cid=C05J3KMPSHE